tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -175,11 +169,31 @@ func TestMain(m *testing.M) {
 		logAndExit(fmt.Sprintf("Mkdir at %q: %v", mntDir, err))
 	}
 
-	ret := m.Run()
+	successCode = m.Run()
 
-	// Delete all files from mntDir to delete files from gcs bucket.
 	os.RemoveAll(mntDir)
 	unMount()
 
-	os.Exit(ret)
+	return
+}
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+
+	if *testBucket == "" {
+		log.Printf("--testbucket must be specified")
+		os.Exit(0)
+	}
+
+	if err := setUpTestDir(); err != nil {
+		log.Printf("setUpTestDir: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Run integration test for go-client library
+	executeTest(true, logFileGoClient, m)
```

Comment:
can you try parameterized tests for this and pass list of flags as parameters.

Reply:
fixed